utils/spacewalk-common-channels.ini

```
@@ -3675,6 +3675,253 @@ checksum = sha256
 base_channels = debian-12-pool-amd64-uyuni
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/Debian12-Uyuni-Client-Tools/Debian_12/
 
+[raspberrypios-12-pool-arm64-uyuni]
+label    = raspberrypios-12-pool-arm64-uyuni
+checksum = sha256
+archs    = arm64-deb
+repo_type = deb
+name     = Raspberry Pi OS 12 (bookworm) main for arm64 for Uyuni
```

```suggestion
name     = Raspberry Pi OS 12 (bookworm) main for ARM64 for Uyuni
```
Maybe?

I saw you used it in all other names.